### PR TITLE
Feature/ch919 docs landing page

### PR DESCRIPTION
**Pull request by sumitkolhe**

### CODE REVIEW

File: src/pages/index.js [hunk 1]
```diff
@@ -1,92 +1,177 @@
...
+              className={styles.infoCardSmall}
+            >
+              <div>
+                <img className={styles.icon} src="/img/code-tags.svg"></img>
+                <h3>Try a Full Stack Example</h3>
+                <p>Deploy PostgreSQL + Hasura + React.</p>
               </div>
+            </Link>
+            <Link
+              style={{ color: "inherit", textDecoration: "inherit" }}
```
Comment (nandiheath) [inline src/pages/index.js:52]: comment: avoid using inline styles if we are using `className` already

File: src/pages/index.js [hunk 2]
```diff
@@ -1,92 +1,177 @@
...
+          </div>
+        </section>
+        <section className={styles.infoCardSection}>
+          <div className={styles.suggestion}>
+            <div className={styles.suggestionText}>
+              <h3>Deploy an Existing Project with Git</h3>
+              <p>
+                Use our Git integrations with
+                <Link
+                  style={{ color: "inherit", textDecoration: "inherit" }}
```
Comment (nandiheath) [inline src/pages/index.js:109]: comment: this one as well

File: src/pages/styles.module.css [hunk 3]
```diff
@@ -1,43 +1,227 @@
...
+  z-index: -1;
+}
+
+.heroBannerRight {
+  height: 312px;
+  position: absolute;
+  right: 0;
+  display: inline-block;
+  z-index: -1;
+}
```
Comment (nandiheath) [inline src/pages/styles.module.css:43]: action: extract the comment attributes to `.heroBanner`

File: src/pages/styles.module.css [hunk 4]
```diff
@@ -1,43 +1,227 @@
...
+  border-radius: 5px;
+  background-color: #1ec4e9;
+}
+.pillYellow {
+  width: 50px;
+  height: 3px;
+  margin: 15px 0;
+  border-radius: 5px;
+  background-color: #f7b500;
+}
```
Comment (nandiheath) [inline src/pages/styles.module.css:83]: action: extract the common fields

File: src/pages/styles.module.css [hunk 5]
```diff
@@ -1,43 +1,227 @@
...
+.cardContentList {
+  list-style-type: none;
+  margin-left: -2rem;
+  font-weight: 600;
+}
+
+.link {
+  color: #bb86fc;
```
Comment (nandiheath) [inline src/pages/styles.module.css:91]: comment: i may consider extracting all the colors as variable (if this is scss)

File: src/pages/styles.module.css [hunk 6]
```diff
@@ -1,43 +1,227 @@
...
+  min-height: 320px;
+  background: var(--ifm-infoCard-background-color);
+  box-shadow: 0px 1px 1px 0px rgba(0, 0, 0, 0.14),
+    0px 2px 1px -1px rgba(0, 0, 0, 0.12), 0px 1px 3px 0px rgba(0, 0, 0, 0.2);
+  border-radius: 2px;
+  margin: 0.5rem 1rem 1rem 1rem;
+  padding: 2rem 2rem 1rem 2rem;
+}
+
+@media screen and (max-width: 990px) {
```
Comment (nandiheath) [inline src/pages/styles.module.css:199]: comment: not sure is it possible to have some breakpoints such as `xs` `sm` so we do not need to provide our own values

File: src/pages/styles.module.css [hunk 7]
```diff
@@ -1,43 +1,227 @@
...
\ No newline at end of file
+.infoCardLarge {
+  width: 360px;
+  height: 330px;
+  min-height: 320px;
+  background: var(--ifm-infoCard-background-color);
+  box-shadow: 0px 1px 1px 0px rgba(0, 0, 0, 0.14),
+    0px 2px 1px -1px rgba(0, 0, 0, 0.12), 0px 1px 3px 0px rgba(0, 0, 0, 0.2);
+  border-radius: 2px;
+  margin: 0.5rem 1rem 1rem 1rem;
```
Comment (nandiheath) [inline src/pages/styles.module.css:195]: question: is there a reason sometimes we use `px` but sometimes `rem`?

File: src/pages/index.js [hunk 8]
```diff
@@ -1,92 +1,177 @@
...
+              </Link>
+            </div>
+            <div className={styles.suggestionText}>
+              <h3>Connect Your Custom Domains</h3>
+              <p>
+                Once deployed, your projects can be assigned to a custom domain.
+              </p>
+
+              <Link
+                style={{ color: "inherit", textDecoration: "inherit" }}
```
Comment (nandiheath) [inline src/pages/index.js:154]: action: inline styles

**Review by nandiheath — CHANGES_REQUESTED** [review]
overall looks good. just some minor css questions there

File: src/pages/index.js (hunk 1 above)
Comment (sumitkolhe) [reply src/pages/index.js:52]: For some reason router Link styles were not getting overriden with css classes so had to use in line. Will check again

**Review by nandiheath — APPROVED** [review]
LGTM

**Pull request merged**